cmake/catkin_package.cmake

```diff
@@ -432,6 +432,31 @@ function(_catkin_package)
   # package cmake dir is the folder where the generated pkgConfig.cmake is located
   set(PKG_CMAKE_DIR "\${${PROJECT_NAME}_DIR}")
 
+  # strip SYSROOT prefixes for each element in LIST
+  macro(catkin_strip_sysroot)
+    cmake_parse_arguments(_fn "UNIQUE" "LIST" "" ${ARGN})
+    set(_list ${${_fn_LIST}})
+    set(${_fn_LIST})
+    foreach(_ele ${_list})
+      if(IS_ABSOLUTE ${_ele})
+        foreach(_path ${CMAKE_FIND_ROOT_PATH})
+          string(REGEX REPLACE "^${_path}" "" _ele ${_ele})
```

Comment:
This is the intended behavior; the result should be again an absolute path, but without the root path. Otherwise the generated pkgConfig.cmake file would see a relative path and would therefore be treated differently in the include logic. However, we want this path to be treated as an absolute path in the target filesystem.